includes/admin/OrderSettings.php

```diff
@@ -603,17 +603,19 @@ private function setInsuranceData(): void
         $isDefaultInsured                  = (bool) $this->getCarrierSetting(WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED);
         $isDefaultInsuredFromPrice         = $this->getCarrierSetting(WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_FROM_PRICE);
         $isDefaultInsuredForBE             = (bool) $this->getCarrierSetting(WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_FOR_BE);
-        $orderTotalExceedsInsuredFromPrice = (float) $this->order->get_total() >= (float) $isDefaultInsuredFromPrice;
+        $orderTotalExceedsInsuredFromPrice = $this->order->get_total() >= (float) $isDefaultInsuredFromPrice;
         $insuranceFromDeliveryOptions      = $this->shipmentOptions->getInsurance();
-        $isBelgium                         = AbstractConsignment::CC_BE === $this->getShippingCountry();
+        $isBe                              = AbstractConsignment::CC_BE === $this->getShippingCountry();
+        $isNl                              = AbstractConsignment::CC_NL === $this->getShippingCountry();
+        $isEu                              = in_array($this->getShippingCountry(), AbstractConsignment::EURO_COUNTRIES) && ! $isNl && ! $isBe;
 
         $carrier             = ConsignmentFactory::createByCarrierName($this->carrier);
-        $amountPossibilities = $carrier->getInsurancePossibilities();
+        $amountPossibilities = $isEu ? $carrier->getEuInsurancePossibilities() : $carrier->getInsurancePossibilities();
```

Comment:
Dan wordt het hier logischer, je kunt nl. $consignment->getInsurancePossibilities(ISO2landcode) doen, volgens de huidige versie van de branch in de sdk waar deze wijzigingen in zitten. Daar is ook het conflict van de protected / public method van getEuInsurancePossibilities() opgelost en je kunt die method dus NIET meer direct aanroepen (is niet meer public).